PKGBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Inexor <https://discord.gg/Sz4SKXpxRN>
+
+buildarch=
+
+pkgname="inexor-git"
+pkgver=0.1.3
+pkgrel=1
+groups=()
+arch=(x86_64)
+pkgdesc=
+url="https://github.com/inexorgame/vulkan-renderer.git"
+license=('MIT')
+depends=('conan' 'cmake' 'ninja' 'make')
```

Comment:
https://wiki.archlinux.org/index.php/PKGBUILD#depends
Your listed software is not required to run the software it should be at `makedepends`.